packages/patternfly-4/content/design-guidelines/usage-and-behavior/pagination/design.md

```diff
@@ -0,0 +1,43 @@
+---
+path: "/design-guidelines/usage-and-behavior/pagination"
+---
+# Pagination
+Pagination is used when you want to break up data into multiple pages. It is typically used within a table, list or card view.  If you choose to use infinite scrolling, then pagination would not be necessary. 
+
+![top-bottom-pagination](./img/top-bottom-pagination.png)
+
+**1. Top pagination:** Pagination is always right-aligned in the toolbar. If pagination is required, we recommend using pagination both above and below content views.
+**2. Bottom pagination:** Pagination should be shown below the content view as well.
+
+## Full pagination
+
+If you do not have to worry about spatial limitations within the top toolbar, we recommend you use full pagination. By adding this to your toolbar, you will have to limit the other actions added in the toolbar on desktop and tablet views. The full variation does condense into only an item count upon mobile.
+
+Desktop full pagination in toolbar
+![full-pagination](./img/full-pagination.png)
```

Comment:
Can we update this screen to show the items per page menu open? That will help illustrate the functionality, I think.